subsys/bluetooth/services/hids.c

```diff
@@ -879,6 +880,11 @@ int bt_hids_init(struct bt_hids *hids_obj,
 {
 	LOG_DBG("Initializing HIDS.");
 
+	if (init_param->rep_map.size > BT_ATT_MAX_ATTRIBUTE_LEN) {
+		LOG_WRN("Report map size exceeds max ATT attribute length");
+		return BT_GATT_ERR(BT_ATT_ERR_INVALID_ATTRIBUTE_LEN);
```

Comment:
Using `BT_GATT_ERR` here is incorrect. The return type of the surrounding function is a negative errno value, like `-EMSGSIZE`. Feel free to use a different errno value if it makes more sense.

```suggestion
		return -EMSGSIZE;
```

Reply:
Change implemented, thanks for the review